sdk/python/approzium/authenticator.py

```diff
@@ -1,14 +1,15 @@
+# needed to be able to import protos code
+import sys
+from pathlib import Path
+
 import approzium
 import grpc
-from pathlib import Path
-from .iam import obtain_signed_get_caller_identity
 
-# needed to be able to import protos code
-import sys
+from .iam import obtain_signed_get_caller_identity
```

Comment:
Question, is this hit by any CI tests? If so, is it hitting the real GetCallerIdentity endpoint? 

I'm _pretty sure_ it's not being hit by tests, but wanted to double-check so we don't get rate-limited by AWS. 😄

Reply:
Thank you for your feedback Becca! I think it does hit the real GetCallerIdentity endpoint. I did not do anything in the tests to "mock" that part or get around it. It would be cool then to check the rate of AWS STS to see if there is an issue there. Good catch!